.github/workflows/inductor-perf-test-nightly-cpu.yml

```diff
@@ -0,0 +1,105 @@
+name: inductor-CPU-perf-nightly
+
+on:
```

Comment:
Could we add cpu nightly test into https://github.com/pytorch/pytorch/blob/main/.github/workflows/inductor-perf-test-nightly.yml also?

Reply:
Perhaps not. 
1. `inductor-perf-test-nightly.yml` will run on `linux.gcp.a100` instance. 
2. All the build and test image are cuda images.
3. [Each A100 instance only has 6-core 2.2GHz Xeon CPU](https://pytorch.org/docs/stable/torch.compiler_performance_dashboard.html)

Reply:
We just need to add cpu build and test as new jobs in that workflow